cmd/juju/commands/bootstrap.go

```diff
@@ -236,7 +236,7 @@ func (c *bootstrapCommand) Init(args []string) (err error) {
 		}
 	}
 	if c.AgentVersion != nil && (c.AgentVersion.Major != jujuversion.Current.Major || c.AgentVersion.Minor != jujuversion.Current.Minor) {
-		return errors.New("requested agent version major.minor mismatch")
+		return errors.Errorf("this client can only bootstrap %v.%v+ agents", jujuversion.Current.Major, jujuversion.Current.Minor)
```

Comment:
Just a thought
2.3+ could imply anything greater than 2.3, eg 2.4
How about
2.3.x

Reply:
Yes, I thought of that but then about about beta versions and such? they take form of, say 2.3-b3...

Maybe just drop suffix altogether? so "...can only bootstrap 2.3 agents"?